infra/base-images/base-runner/bad_build_check

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash -u
```

Comment:
you also need to add this to the COPY command in the dockerfile: https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-runner/Dockerfile